### PR TITLE
feat(DotSteps): update component dots style

### PR DESCRIPTION
**Pull request by adriengibrat**
## Type

Enhancement

### Summarise concisely:

Update the DotSteps styles

#### What is expected?

Update the new selected / active dot style.

#### The following changes where made:

1. Use aria-selected to mark selected dot

2. Use `styled` to customize dot styles

3. Update test snapshots

## Relevant logs and/or screenshots

Before
image

After
image

### CODE REVIEW

**Review by chambo-e — COMMENTED** [review]
I think we miss the animation 🙂 


https://user-images.githubusercontent.com/2802867/131560071-c73e7ced-d8e5-4274-8876-09ff892ee088.mov

**Comment by adriengibrat** [comment]
With animation!

https://user-images.githubusercontent.com/62407/131717472-5cbf8770-be49-416c-9134-8b3176331934.mp4

**Review by vincentaudebert — APPROVED** [review]

**Comment by DorianMaliszewski** [comment]
Code cov of Dot decrease by 14%, please can you make 100% again ? 🙏

**Review by chambo-e — COMMENTED** [review]
Seems like we decreased the coverage of Dot by not using color anymore, could you add a simple test in Dot pls ? :)

File: src/components/DotSteps/index.tsx [hunk 1]
```diff
@@ -15,19 +68,15 @@ const DotSteps: FunctionComponent<Props> = ({
   step = 1,
   setStep,
 }) => (
-  <FlexBox justifyContent="center">
+  <DotWrapper justifyContent="center" data-steps={steps}>
```
[the remark this answers is not in the document]
Comment (adriengibrat) [reply src/components/DotSteps/index.tsx]: done (amended cfce5a8)

**Review by chambo-e — APPROVED** [review]

**Review by DorianMaliszewski — APPROVED** [review]

**Pull request merged**